src/index.ts

```diff
@@ -68,7 +60,7 @@ class Storyblok {
   private client: SbFetch;
   private maxRetries: number;
   private retriesDelay: number;
-  private throttle: ThrottleFn;
+  private throttle;
```

Comment:
Given this is assigned the return value of `throttledQueue()` on line 143, I think we can type this as following:

```suggestion
  private throttle: ReturnType<typeof throttledQueue>;
```

Reply:
💯